cultural-fund-map/main.js

```diff
@@ -1,8 +1,51 @@
-var map = L.map('map').setView([39.967, -75.237], 13);
+var map = L.map('map').setView([39.949669, -75.164361], 13);
 
 L.tileLayer('https://api.tiles.mapbox.com/v4/{id}/{z}/{x}/{y}.png?access_token={accessToken}', {
     attribution: 'Map data &copy; <a href="http://openstreetmap.org">OpenStreetMap</a> contributors, <a href="http://creativecommons.org/licenses/by-sa/2.0/">CC-BY-SA</a>, Imagery © <a href="http://mapbox.com">Mapbox</a>',
     maxZoom: 18,
     id: 'cahdeemer.n2p8d0mj',
     accessToken: 'pk.eyJ1IjoiY2FoZGVlbWVyIiwiYSI6ImIwYzM3OGYyZTBmYmZlMWJmMzQ5OTNmZWRjMTA3NjNmIn0.4lMKKySybu846ym7-BNbYA'
-}).addTo(map);
\ No newline at end of file
+}).addTo(map);
+
+var dataSuccess = function( jsonData ) {
+	console.log(jsonData);
+	var layerOptions = {
+		pointToLayer: function(featureData, latlng) {
+			var markerOptions = {
+				color: '#f1f2f2',
+				fillOpacity: 0.9,
+				fillColor: (function(x) {
+					if (x > 49999)
```

Comment:
Although Javascript doesn't enforce curly braces around an `if` block, it's encouraged because it helps reduce errors if you add additional lines to that conditional, also you can use an `if else` block to guard against multiple comparisons without having to have multiple return statements.  I find it easier to reason about a function if it exists in one place, usually at the end.  Consider using something like this:

```javascript
var color = '#99c5c3';

if (x > 49999) {
    color = '#002C35';
} else if (x > 24999) {
    color = '#006e85';
} else if (x > 14999) {
    color = '#4d9aaa';
}

return color;
```

Note that by assigning a default value to the initial `color` variable, you don't need to do an additional check if the value fails all checks.

This is functionally equivalent to what you wrote, but I find it ends up in code that is easier to maintain and extend.

Also, because the order of the conditionals are important (the second check is logically true if the first is) I would put a comment to yourself or others that the order of the value checks should be high to low to prevent tricky bugs later.